examples/launch_huggingface_classification.py

```diff
@@ -77,7 +77,9 @@
         config_space, mode=mode, metric=metric, random_seed=random_seed
     )
 
-    stop_criterion = StoppingCriterion(max_wallclock_time=3000) # wall clock time can be increased to 1 hour for more performance
+    stop_criterion = StoppingCriterion(
```

Comment:
The previous PR must not have run black properly

Reply:
That was me - seems like it's not captured in our package linter rule, I will take a look next week

Reply:
Thanks for flagging this, my miss